src/main/java/org/dependencytrack/persistence/ProjectQueryFilterBuilder.java

```diff
@@ -71,6 +72,18 @@ ProjectQueryFilterBuilder withFuzzyNameOrExactTag(String name, Tag tag) {
         return this;
     }
 
+    ProjectQueryFilterBuilder excludeProjectsThatAreChildren(){
+        filterCriteria.add("parent == null");
+        return this;
+    }
+
+    ProjectQueryFilterBuilder withParent(UUID uuid){
+        params.put("uuid", uuid);
+
+        filterCriteria.add("parent.uuid == :uuid");
```

Comment:
Using `:parentUuid` would be better to avoid any conflicts if the same query has a `:uuid` parameter already.